src/views/ipv/page/pyi-update-details.njk

```diff
@@ -0,0 +1,109 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+
+{% set pageTitleKey = 'pages.pyiUpdateDetails.title' %}
+{% set googleTagManagerPageId = "pyiUpdateDetails" %}
+
+{% set errorState = pageErrorState %}
+{% set errorTitle = 'pages.pyiUpdateDetails.content.formErrorMessage.errorSummaryTitleText' | translate %}
+{% set errorText = 'pages.pyiUpdateDetails.content.formErrorMessage.errorSummaryDescriptionText' | translate %}
+
+{% block content %}
+    <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}"
+        xmlns="http://www.w3.org/1999/html">{{ 'pages.pyiUpdateDetails.header' | translate }}</h1>
+
+    {% set rows = [
+      {
+        key: {
+          text: 'pages.pyiUpdateDetails.content.summaryListHeading1' | translate
+        },
+        value: {
+          text: userDetails.formattedName.GivenName
+        }
+      },
+      {
+        key: {
+          text: 'pages.pyiUpdateDetails.content.summaryListHeading2' | translate
+        },
+        value: {
+          text: userDetails.formattedName.FamilyName
+        }
+      },
+      {
+          key: {
+              text: 'pages.pyiUpdateDetails.content.summaryListHeading3' | translate
+          },
+          value: {
+              text: userDetails.addresses[0].addressDetailHtml | safe
+          }
+      },
+      {
+          key: {
+            text: 'pages.pyiUpdateDetails.content.summaryListHeading4' | translate
+          },
+          value: {
+            text: userDetails.dateOfBirth | GDSDate
+          }
+      }
+    ] %}
+
+    {{ govukSummaryList({
+      rows: rows
+    }) }}
+
+    <form id="confirmAddressActionForm" action="/ipv/page/{{ pageId }}" method="POST">
+            <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+            {% set checkboxConfig = {
+              name: "nationality",
+              fieldset: {
+                legend: {
+                  text: 'pages.pyiUpdateDetails.content.checkboxes.heading' | translate,
+                  isPageHeading: true,
```

Comment:
I think we don't want `isPageHeading: true`, the design docs suggest that it should be set to true if the legend also acts as the heading for the page.

I would assume that this isn't the case given we have an h1 header above it in the page?

Reply:
(We should only be using this if we want to avoid the screenreader from repeating the same heading twice, if the heading of the page is the same as the radio button header)

Reply:
Oh, didn't know that. Good to know 👍